src/components/TodoList/TodoList.tsx

```diff
@@ -29,26 +31,33 @@ const TodoList = () => {
   };
 
   return (
-    <List>
-      {todos.map((todo: Todo) => (
-        <ListItem key={todo.id}>
-          <Checkbox
-            edge='start'
-            checked={todo.completed}
-            tabIndex={-1}
-            onClick={() => handleToggle(todo.id)}
-          />
-          <ListItemText
-            primary={todo.text}
-            style={{ textDecoration: todo.completed ? 'line-through' : 'none' }}
-          />
-          <ListItemSecondaryAction>
-            <IconButton edge='end' onClick={() => handleDelete(todo.id)}>
-              <DeleteIcon />
-            </IconButton>
-          </ListItemSecondaryAction>
-        </ListItem>
-      ))}
+    <List sx={{width: '100%', maxHeight: '50vh', overflowY: 'auto'}}>
```

Comment:
Shouldn't these be defined in `makeStyles`? Inline changes should be used only if you need to set dynamic styles

Reply:
In the Material UI docs, the 'makeStyles' function has a Legacy mark:
https://mui.com/system/styles/basics/
And 'sx' function is presented as the main method to add CSS styles in JS:
https://mui.com/system/getting-started/the-sx-prop/
Does it make sense for you?